### ISSUE TITLE
support for Serilog 2

### ISSUE THREAD

**Issue by matt-softlogic** [serilog/serilog-sinks-elasticsearch#45]
Is it on the cards, and if so, what's the plan for Serilog 2.0 support?  We're migrating to the ELK stack and have a code base on SL2 already, and would like to continue to stay on that release.

Thanks in advance

**Comment by nblumhardt** [comment]
Hi Matt - there's some discussion in #21, probably the best thread to track this on currently. Cheers!

**Comment by mbarragan** [comment]
Is there any update on this issue? I'm using _Serilog 2.0.0_ and I can't use the elasticsearch sink because it only supports _Serilog (>= 1.5.14 && < 2.0.0)_.

**Comment by nblumhardt** [comment]
I'm not sure about the status of .NET Core but it should be quite straightforward to do an update for this package that works on the full framework and targets Serilog 2.0. Are you on core, or full? Thanks!

**Comment by mbarragan** [comment]
Currently I'm i'm using AspNetCore and .NET4.6.1

**Comment by nblumhardt** [comment]
Okay, this should work with a straight csproj upgrade. I am on vacation this week, but in case anyone can send a PR the instructions are:

```
Update-Package Serilog
Install-Package Serilog.Sinks.PeriodicBatching
```

There will then be a few issues from the [Upgrade Guide](https://github.com/serilog/serilog/wiki/2.x-Upgrade-Guide) to take care of, but shouldn't be anything tricky.

The only potential breaking change is removal of .NET 4 support; bumping the major version in AppVeyor should take care of this.

@serilog/reviewers-elasticsearch are you guys happy for someone to pick this bit off? Cheers! :-)

**Comment by chuwik** [comment]
Hi guys, I gave it a try in #53. Hope it helps!

**Comment by nblumhardt** [comment]
Closed in #53.
